src/funding-address.ts

```diff
@@ -0,0 +1,32 @@
+'use strict'
```

Comment:
You should look up what `use strict` does in TypeScript - Answer: Nothing. In fact, for recent versions of TypeScript, it's _added_ to the Javascript output all the time. So... you might want to consider yanking these in a batch.

Just consider.